testing/internal/e2e/boundary/scope.go

```diff
@@ -83,3 +93,29 @@ func CreateNewProjectCli(t testing.TB, ctx context.Context, orgId string) string
 	t.Logf("Created Project Id: %s", newProjectId)
 	return newProjectId
 }
+
+// getScopeOpts iterates the inbound ScopeOptions and returns a struct
+func getScopeOpts(opt ...ScopeOption) scopeOptions {
```

Comment:
For targets, we use boundary's target options already defined in the application code. Is there not a corresponding one for scopes?

Reply:
I haven't found it in the code base. If I missed that, let me know, please 🙏

Reply:
I'm not super familiar with this. I did find this:https://github.com/hashicorp/boundary-enterprise/blob/main/api/scopes/option.gen.go?

Reply:
Hm. I might be confusing them with the API options (https://github.com/hashicorp/boundary-enterprise/blob/main/api/scopes/option.gen.go).